music21/articulations.py

```diff
@@ -582,10 +583,27 @@ class PullOff(spanner.Spanner, TechnicalIndication):
     pass
 
 class FretBend(FretIndication):
-    bendAlter: interval.IntervalBase | None = None
-    preBend: t.Any = None
-    release: t.Any = None
-    withBar: t.Any = None
+    bendAlter: interval.IntervalBase
+    preBend: bool = False
+    release: t.Optional[float]
+    withBar: t.Optional[bool]
+
+    def __init__(self, number=0, preBend=False, release=None, withBar=None, **keywords):
+        '''
+        bend indication for fretted instruments
+
+        bend in musicxml
+
+        number is the interval of the bend in number of semitones, bend-alter in musicxml
+        preBend indicates wether the note is prebended or not
+        release is the offset value for releasing the bend, if Any
+        withBar indicates if the bend is done using a whammy bar movement
```

Comment:
Capitalization and use asterisks and line breaks for documentation -- run documentation/build.py and look at the difference between the docs produced here and elsewhere in the system.

`preBend indicates wether the note is prebended or not` -- I don't know what prebended means.

what is release measured in?  Is it an offset (which is relative to the start of the Measure/stream)?  That doesn't sound right.  Is it a quarterLength measured from the start of the note? or from the end of the note.

withBar doc is fine.

Reply:
I've tried to improve it in https://github.com/cuthbertLab/music21/pull/1580/commits/03e6117e967d371851f8c80e73c3b2218a2ebf06 
Let me know if that seems good to you